mmv1/products/integrationconnectors/Connection.yaml

```diff
@@ -0,0 +1,1120 @@
+  - !ruby/object:Api::Type::NestedObject
+    name: "authConfig"
+    description: |
+      authConfig for the connection.
+    properties:
+      - !ruby/object:Api::Type::Enum
+        name: "authType"
+        description: |
+          authType of the Connection
+        required: true
+        values:
+          - USER_PASSWORD
+          - OAUTH2_JWT_BEARER
+          - OAUTH2_CLIENT_CREDENTIALS
+          - SSH_PUBLIC_KEY
+          - OAUTH2_AUTH_CODE_FLOW
+      - !ruby/object:Api::Type::Array
+        name: "additionalVariable"
+        api_name: "additionalVariables"
+        description: |
+          List containing additional auth configs.
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: "key"
+              description: |
+                Key for the configVariable
+              required: true
+            - !ruby/object:Api::Type::Integer
+              name: "integerValue"
+              api_name: "intValue"
+              description: |
+                Integer Value of configVariable.
+            - !ruby/object:Api::Type::String
+              name: "stringValue"
+              description: |
+                String Value of configVariabley.
+            - !ruby/object:Api::Type::Boolean
+              name: "booleanValue"
+              api_name: "boolValue"
+              description: |
+                Boolean Value of configVariable.
+            - !ruby/object:Api::Type::NestedObject
+              name: "secretValue"
+              description: |
+                Secret value of configVariable.
+              properties:
+                - !ruby/object:Api::Type::String
+                  name: "secretVersion"
+                  required: true
+                  description: |
+                    Secret version of Secret Value for Config variable.
+            - !ruby/object:Api::Type::NestedObject
+              name: "encryptionKeyValue"
+              description: |
+                Encription key value of configVariable.
+              properties:
+                - !ruby/object:Api::Type::Enum
+                  name: "type"
+                  description: |
+                    Type of Encription Key
+                  values:
+                    - GOOGLE_MANAGED
+                    - CUSTOMER_MANAGED
+                - !ruby/object:Api::Type::String
+                  name: "kmsKeyName"
+                  description: |
+                    The [KMS key name] with which the content of the Operation is encrypted. The expected
+                    format: projects/*/locations/*/keyRings/*/cryptoKeys/*.
+                    Will be empty string if google managed.
+      - !ruby/object:Api::Type::String
+        name: "authKey"
+        description: |
+          The type of authentication configured.
+      - !ruby/object:Api::Type::NestedObject
+        name: "userPassword"
```

Comment:
need description for this field. I understand that the field is kind of speak for itself, but we need the description for web doc, otherwise it will generated a placeholder for the field. See https://github.com/modular-magician/terraform-provider-google-beta/blob/73f1e7f142c6eff78ddb2fc00205355d86265095/website/docs/r/integration_connectors_connection.html.markdown?plain=1#L294

Reply:
Added